app/assets/javascripts/discourse/app/components/admin-post-menu.gjs

```diff
@@ -60,45 +68,45 @@ export default class AdminPostMenu extends Component {
         </li>
       {{/if}}
 
-      {{#if (and this.currentUser.staff (not @data.transformedPost.isWhisper))}}
+      {{#if (and this.currentUser.staff (not @transformedPost.isWhisper))}}
```

Comment:
```suggestion
      {{#if (and this.currentUser.staff (not this.transformedPost.isWhisper))}}
```
?

Reply:
(and quite a few others in the file I think)

Reply:
weird 🤔  but yes